### PR TITLE
Modernize libva-utils workflow

### PR DESCRIPTION
**Pull request by ehfd**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

**Comment by hmaarrfk** [comment]
hmm there are typically challeninges in switching from automake to cmake, but this isn't a library, so OK!

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@ECHO ON
+
+:: set pkg-config path so that host deps can be found
+:: (set as env var so it's used by meson)
+set "PKG_CONFIG_PATH=%LIBRARY_LIB%\pkgconfig;%LIBRARY_PREFIX%\share\pkgconfig;%BUILD_PREFIX%\Library\lib\pkgconfig"
+
+:: get mixed path (forward slash) form of prefix so host prefix replacement works
+set "LIBRARY_PREFIX_M=%LIBRARY_PREFIX:\=/%"
+
+%BUILD_PREFIX%\Scripts\meson.exe setup builddir --buildtype=release --prefix=%LIBRARY_PREFIX_M% --backend=ninja
```
Comment (hmaarrfk) [inline recipe/bld.bat:10]:
```suggestion
%BUILD_PREFIX%\Scripts\meson.exe setup builddir %MESON_ARGS%
```

did MESON_ARGS not work?

**Review by hmaarrfk — COMMENTED** [review]
small question

File: recipe/bld.bat (hunk 1 above)
Comment (ehfd) [reply recipe/bld.bat:10]: Time to try. It did not work for a long time.

Comment (hmaarrfk) [reply recipe/bld.bat:10]: correct. i just feel like recently it has bee ngetting more and more defined.

Comment (ehfd) [reply recipe/bld.bat:10]: @hmaarrfk It fails precisely related to that part.

**Comment by ehfd** [comment]
@hmaarrfk Windows build seems to want DirectX Headers. Is there a feedstock for this?

**Comment by hmaarrfk** [comment]
i'm not sure sorry

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

**Comment by ehfd** [comment]
I am excluding Windows for now because it leads to foreign files. Once the DirectX-Headers feedstock exists it can be added any time.

**Pull request merged**